src/libraries/System.Text.Json/tests/Serialization/ExtensionDataTests.cs

```diff
@@ -770,12 +770,16 @@ private class ClassWithExtensionPropertyCustomIImmutableJsonElement
             public GenericIImmutableDictionaryWrapper<string, JsonElement> MyOverflow { get; set; }
         }
 
-        [Fact]
-        public static void DeserializeIntoGenericDictionaryParameterCount()
+        [Theory]
```

Comment:
This change is a follow up to https://github.com/dotnet/runtime/pull/33014#pullrequestreview-367494573,